.github/actions/audit-accessibility/main.js

```diff
@@ -30,9 +30,6 @@ process.on('unhandledRejection', (error) => {
 
 const getStories = () => {
     console.log('Extracting stories information');
-    // creates "stories.json" file with stories information
```

Comment:
This is not neccesary in Storybook 7, because `stories.json` is created automatically when building storybook.